test_scripts/API/AddCommand/SDL_resends_WARNINGS_to_MOB_when_HMI_respond_WARNINGS_to_one_HMI_portions.txt

```diff
@@ -0,0 +1,14 @@
+./test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_TTS_SetGlobalProperties_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_SetGlobalProperties
```

Comment:
@vvvakulenko, According to attached question:  APPLINK-15210 "Need to confirm SDL behavior in case HMI respond with erroneous result code at least to one component of RPC"
More scripts should be added.